redis_feature_store.js

```diff
@@ -1,60 +1,38 @@
-const redis = require('redis'),
-  dataKind = require('launchdarkly-node-server-sdk/versioned_data_kind'),
-  CachingStoreWrapper = require('launchdarkly-node-server-sdk/caching_store_wrapper');
+const base = require('./redis_base');
+const dataKind = require('launchdarkly-node-server-sdk/versioned_data_kind');
+const CachingStoreWrapper = require('launchdarkly-node-server-sdk/caching_store_wrapper');
 
 const noop = function() {};
 
-function RedisFeatureStore(redisOpts, cacheTTL, prefix, logger, preconfiguredClient) {
+function RedisFeatureStore(redisOpts, cacheTTL, prefix, logger, client) {
+  let options;
+  if (cacheTTL || prefix || logger || client) {
+    // convert from older syntax
+    options = { redisOpts, cacheTTL, prefix, client };
```

Comment:
Doesn't this and line 17 need to specify field names for the options object? I think it would also makes sense to include `logger` here and use `options.logger || config.logger` on line 22.

Reply:
The syntax `{ a, b, c }` is simply a shorthand for `{ a: a, b: b, c: c }`. In other words, the field names are omitted here because they are the same as the names of the parameters they're being set from. That's the same as how for instance the `module.exports` properties are being set in `redis_big_segment_store.js`.

Reply:
Ah, I clearly don't do that much JS.